[2024-12-06T22:19:01.790-0600] {processor.py:186} INFO - Started process (PID=12505) to work on /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T22:19:01.792-0600] {processor.py:914} INFO - Processing file /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-06T22:19:01.796-0600] {logging_mixin.py:190} INFO - [2024-12-06T22:19:01.795-0600] {dagbag.py:588} INFO - Filling up the DagBag from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T22:19:01.803-0600] {logging_mixin.py:190} INFO - [2024-12-06T22:19:01.802-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-06T22:19:01.804-0600] {logging_mixin.py:190} INFO - [2024-12-06T22:19:01.803-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-06T22:19:01.804-0600] {processor.py:927} WARNING - No viable dags retrieved from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T22:19:01.818-0600] {processor.py:208} INFO - Processing /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.040 seconds
[2024-12-06T22:20:02.364-0600] {processor.py:186} INFO - Started process (PID=12616) to work on /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T22:20:02.372-0600] {processor.py:914} INFO - Processing file /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-06T22:20:02.375-0600] {logging_mixin.py:190} INFO - [2024-12-06T22:20:02.374-0600] {dagbag.py:588} INFO - Filling up the DagBag from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T22:20:02.382-0600] {logging_mixin.py:190} INFO - [2024-12-06T22:20:02.380-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-06T22:20:02.382-0600] {logging_mixin.py:190} INFO - [2024-12-06T22:20:02.382-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-06T22:20:02.383-0600] {processor.py:927} WARNING - No viable dags retrieved from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T22:20:02.403-0600] {processor.py:208} INFO - Processing /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.051 seconds
[2024-12-06T22:20:57.284-0600] {processor.py:186} INFO - Started process (PID=12698) to work on /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T22:20:57.291-0600] {processor.py:914} INFO - Processing file /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-06T22:20:57.295-0600] {logging_mixin.py:190} INFO - [2024-12-06T22:20:57.295-0600] {dagbag.py:588} INFO - Filling up the DagBag from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T22:20:57.302-0600] {logging_mixin.py:190} INFO - [2024-12-06T22:20:57.301-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-06T22:20:57.303-0600] {logging_mixin.py:190} INFO - [2024-12-06T22:20:57.302-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-06T22:20:57.303-0600] {processor.py:927} WARNING - No viable dags retrieved from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T22:20:57.323-0600] {processor.py:208} INFO - Processing /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.051 seconds
[2024-12-06T22:21:55.505-0600] {processor.py:186} INFO - Started process (PID=12773) to work on /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T22:21:55.506-0600] {processor.py:914} INFO - Processing file /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-06T22:21:55.508-0600] {logging_mixin.py:190} INFO - [2024-12-06T22:21:55.508-0600] {dagbag.py:588} INFO - Filling up the DagBag from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T22:21:55.512-0600] {logging_mixin.py:190} INFO - [2024-12-06T22:21:55.511-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-06T22:21:55.512-0600] {logging_mixin.py:190} INFO - [2024-12-06T22:21:55.512-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-06T22:21:55.512-0600] {processor.py:927} WARNING - No viable dags retrieved from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T22:21:55.521-0600] {processor.py:208} INFO - Processing /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2024-12-06T22:23:13.083-0600] {processor.py:186} INFO - Started process (PID=12937) to work on /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T22:23:13.090-0600] {processor.py:914} INFO - Processing file /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-06T22:23:13.092-0600] {logging_mixin.py:190} INFO - [2024-12-06T22:23:13.091-0600] {dagbag.py:588} INFO - Filling up the DagBag from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T22:23:13.094-0600] {logging_mixin.py:190} INFO - [2024-12-06T22:23:13.094-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-06T22:23:13.095-0600] {logging_mixin.py:190} INFO - [2024-12-06T22:23:13.094-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-06T22:23:13.095-0600] {processor.py:927} WARNING - No viable dags retrieved from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T22:23:13.103-0600] {processor.py:208} INFO - Processing /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2024-12-06T22:24:09.951-0600] {processor.py:186} INFO - Started process (PID=13007) to work on /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T22:24:09.957-0600] {processor.py:914} INFO - Processing file /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-06T22:24:09.959-0600] {logging_mixin.py:190} INFO - [2024-12-06T22:24:09.959-0600] {dagbag.py:588} INFO - Filling up the DagBag from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T22:24:09.961-0600] {logging_mixin.py:190} INFO - [2024-12-06T22:24:09.961-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-06T22:24:09.962-0600] {logging_mixin.py:190} INFO - [2024-12-06T22:24:09.961-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-06T22:24:09.962-0600] {processor.py:927} WARNING - No viable dags retrieved from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T22:24:09.970-0600] {processor.py:208} INFO - Processing /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2024-12-06T22:25:07.032-0600] {processor.py:186} INFO - Started process (PID=13092) to work on /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T22:25:07.039-0600] {processor.py:914} INFO - Processing file /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-06T22:25:07.040-0600] {logging_mixin.py:190} INFO - [2024-12-06T22:25:07.040-0600] {dagbag.py:588} INFO - Filling up the DagBag from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T22:25:07.044-0600] {logging_mixin.py:190} INFO - [2024-12-06T22:25:07.043-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-06T22:25:07.044-0600] {logging_mixin.py:190} INFO - [2024-12-06T22:25:07.044-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-06T22:25:07.045-0600] {processor.py:927} WARNING - No viable dags retrieved from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T22:25:07.058-0600] {processor.py:208} INFO - Processing /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.032 seconds
[2024-12-06T22:26:03.042-0600] {processor.py:186} INFO - Started process (PID=13173) to work on /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T22:26:03.043-0600] {processor.py:914} INFO - Processing file /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-06T22:26:03.044-0600] {logging_mixin.py:190} INFO - [2024-12-06T22:26:03.044-0600] {dagbag.py:588} INFO - Filling up the DagBag from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T22:26:03.047-0600] {logging_mixin.py:190} INFO - [2024-12-06T22:26:03.047-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-06T22:26:03.047-0600] {logging_mixin.py:190} INFO - [2024-12-06T22:26:03.047-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-06T22:26:03.048-0600] {processor.py:927} WARNING - No viable dags retrieved from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T22:26:03.057-0600] {processor.py:208} INFO - Processing /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.020 seconds
[2024-12-06T22:27:00.153-0600] {processor.py:186} INFO - Started process (PID=13252) to work on /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T22:27:00.160-0600] {processor.py:914} INFO - Processing file /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-06T22:27:00.161-0600] {logging_mixin.py:190} INFO - [2024-12-06T22:27:00.161-0600] {dagbag.py:588} INFO - Filling up the DagBag from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T22:27:00.165-0600] {logging_mixin.py:190} INFO - [2024-12-06T22:27:00.165-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-06T22:27:00.165-0600] {logging_mixin.py:190} INFO - [2024-12-06T22:27:00.165-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-06T22:27:00.166-0600] {processor.py:927} WARNING - No viable dags retrieved from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T22:27:00.180-0600] {processor.py:208} INFO - Processing /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.032 seconds
[2024-12-06T22:37:20.789-0600] {processor.py:186} INFO - Started process (PID=14639) to work on /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T22:37:20.796-0600] {processor.py:914} INFO - Processing file /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-06T22:37:20.800-0600] {logging_mixin.py:190} INFO - [2024-12-06T22:37:20.799-0600] {dagbag.py:588} INFO - Filling up the DagBag from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T22:37:20.807-0600] {logging_mixin.py:190} INFO - [2024-12-06T22:37:20.806-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-06T22:37:20.807-0600] {logging_mixin.py:190} INFO - [2024-12-06T22:37:20.807-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-06T22:37:20.808-0600] {processor.py:927} WARNING - No viable dags retrieved from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T22:37:20.826-0600] {processor.py:208} INFO - Processing /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.050 seconds
[2024-12-06T22:38:19.005-0600] {processor.py:186} INFO - Started process (PID=14744) to work on /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T22:38:19.006-0600] {processor.py:914} INFO - Processing file /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-06T22:38:19.008-0600] {logging_mixin.py:190} INFO - [2024-12-06T22:38:19.007-0600] {dagbag.py:588} INFO - Filling up the DagBag from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T22:38:19.010-0600] {logging_mixin.py:190} INFO - [2024-12-06T22:38:19.010-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-06T22:38:19.011-0600] {logging_mixin.py:190} INFO - [2024-12-06T22:38:19.011-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-06T22:38:19.011-0600] {processor.py:927} WARNING - No viable dags retrieved from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T22:38:19.019-0600] {processor.py:208} INFO - Processing /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.019 seconds
[2024-12-06T22:39:17.219-0600] {processor.py:186} INFO - Started process (PID=14879) to work on /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T22:39:17.222-0600] {processor.py:914} INFO - Processing file /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-06T22:39:17.229-0600] {logging_mixin.py:190} INFO - [2024-12-06T22:39:17.228-0600] {dagbag.py:588} INFO - Filling up the DagBag from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T22:39:17.237-0600] {logging_mixin.py:190} INFO - [2024-12-06T22:39:17.236-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-06T22:39:17.237-0600] {logging_mixin.py:190} INFO - [2024-12-06T22:39:17.237-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-06T22:39:17.238-0600] {processor.py:927} WARNING - No viable dags retrieved from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T22:39:17.261-0600] {processor.py:208} INFO - Processing /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.074 seconds
[2024-12-06T22:40:17.047-0600] {processor.py:186} INFO - Started process (PID=15044) to work on /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T22:40:17.053-0600] {processor.py:914} INFO - Processing file /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-06T22:40:17.056-0600] {logging_mixin.py:190} INFO - [2024-12-06T22:40:17.055-0600] {dagbag.py:588} INFO - Filling up the DagBag from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T22:40:17.059-0600] {logging_mixin.py:190} INFO - [2024-12-06T22:40:17.058-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-06T22:40:17.059-0600] {logging_mixin.py:190} INFO - [2024-12-06T22:40:17.059-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-06T22:40:17.059-0600] {processor.py:927} WARNING - No viable dags retrieved from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T22:40:17.067-0600] {processor.py:208} INFO - Processing /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2024-12-06T22:41:19.297-0600] {processor.py:186} INFO - Started process (PID=15165) to work on /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T22:41:19.304-0600] {processor.py:914} INFO - Processing file /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-06T22:41:19.307-0600] {logging_mixin.py:190} INFO - [2024-12-06T22:41:19.307-0600] {dagbag.py:588} INFO - Filling up the DagBag from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T22:41:19.314-0600] {logging_mixin.py:190} INFO - [2024-12-06T22:41:19.313-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-06T22:41:19.314-0600] {logging_mixin.py:190} INFO - [2024-12-06T22:41:19.314-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-06T22:41:19.315-0600] {processor.py:927} WARNING - No viable dags retrieved from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T22:41:19.330-0600] {processor.py:208} INFO - Processing /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.046 seconds
[2024-12-06T23:25:27.912-0600] {processor.py:186} INFO - Started process (PID=17291) to work on /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T23:25:27.918-0600] {processor.py:914} INFO - Processing file /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-06T23:25:27.920-0600] {logging_mixin.py:190} INFO - [2024-12-06T23:25:27.919-0600] {dagbag.py:588} INFO - Filling up the DagBag from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T23:25:27.922-0600] {logging_mixin.py:190} INFO - [2024-12-06T23:25:27.922-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-06T23:25:27.923-0600] {logging_mixin.py:190} INFO - [2024-12-06T23:25:27.923-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-06T23:25:27.923-0600] {processor.py:927} WARNING - No viable dags retrieved from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T23:25:27.932-0600] {processor.py:208} INFO - Processing /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2024-12-06T23:26:25.852-0600] {processor.py:186} INFO - Started process (PID=17402) to work on /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T23:26:25.859-0600] {processor.py:914} INFO - Processing file /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-06T23:26:25.861-0600] {logging_mixin.py:190} INFO - [2024-12-06T23:26:25.861-0600] {dagbag.py:588} INFO - Filling up the DagBag from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T23:26:25.868-0600] {logging_mixin.py:190} INFO - [2024-12-06T23:26:25.867-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-06T23:26:25.869-0600] {logging_mixin.py:190} INFO - [2024-12-06T23:26:25.868-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-06T23:26:25.869-0600] {processor.py:927} WARNING - No viable dags retrieved from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T23:26:25.885-0600] {processor.py:208} INFO - Processing /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.044 seconds
[2024-12-06T23:27:23.012-0600] {processor.py:186} INFO - Started process (PID=17475) to work on /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T23:27:23.018-0600] {processor.py:914} INFO - Processing file /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-06T23:27:23.020-0600] {logging_mixin.py:190} INFO - [2024-12-06T23:27:23.020-0600] {dagbag.py:588} INFO - Filling up the DagBag from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T23:27:23.026-0600] {logging_mixin.py:190} INFO - [2024-12-06T23:27:23.025-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-06T23:27:23.026-0600] {logging_mixin.py:190} INFO - [2024-12-06T23:27:23.026-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-06T23:27:23.026-0600] {processor.py:927} WARNING - No viable dags retrieved from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T23:27:23.033-0600] {processor.py:208} INFO - Processing /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.029 seconds
[2024-12-06T23:28:22.050-0600] {processor.py:186} INFO - Started process (PID=17578) to work on /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T23:28:22.057-0600] {processor.py:914} INFO - Processing file /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-06T23:28:22.059-0600] {logging_mixin.py:190} INFO - [2024-12-06T23:28:22.058-0600] {dagbag.py:588} INFO - Filling up the DagBag from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T23:28:22.063-0600] {logging_mixin.py:190} INFO - [2024-12-06T23:28:22.062-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-06T23:28:22.063-0600] {logging_mixin.py:190} INFO - [2024-12-06T23:28:22.063-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-06T23:28:22.064-0600] {processor.py:927} WARNING - No viable dags retrieved from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T23:28:22.074-0600] {processor.py:208} INFO - Processing /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.033 seconds
[2024-12-06T23:29:17.167-0600] {processor.py:186} INFO - Started process (PID=17722) to work on /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T23:29:17.173-0600] {processor.py:914} INFO - Processing file /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-06T23:29:17.174-0600] {logging_mixin.py:190} INFO - [2024-12-06T23:29:17.174-0600] {dagbag.py:588} INFO - Filling up the DagBag from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T23:29:17.179-0600] {logging_mixin.py:190} INFO - [2024-12-06T23:29:17.178-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-06T23:29:17.179-0600] {logging_mixin.py:190} INFO - [2024-12-06T23:29:17.179-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-06T23:29:17.179-0600] {processor.py:927} WARNING - No viable dags retrieved from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T23:29:17.191-0600] {processor.py:208} INFO - Processing /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.032 seconds
[2024-12-06T23:30:14.465-0600] {processor.py:186} INFO - Started process (PID=17811) to work on /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T23:30:14.471-0600] {processor.py:914} INFO - Processing file /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-06T23:30:14.472-0600] {logging_mixin.py:190} INFO - [2024-12-06T23:30:14.471-0600] {dagbag.py:588} INFO - Filling up the DagBag from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T23:30:14.474-0600] {logging_mixin.py:190} INFO - [2024-12-06T23:30:14.474-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-06T23:30:14.474-0600] {logging_mixin.py:190} INFO - [2024-12-06T23:30:14.474-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-06T23:30:14.474-0600] {processor.py:927} WARNING - No viable dags retrieved from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T23:30:14.482-0600] {processor.py:208} INFO - Processing /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2024-12-06T23:31:09.649-0600] {processor.py:186} INFO - Started process (PID=17916) to work on /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T23:31:09.655-0600] {processor.py:914} INFO - Processing file /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-06T23:31:09.657-0600] {logging_mixin.py:190} INFO - [2024-12-06T23:31:09.656-0600] {dagbag.py:588} INFO - Filling up the DagBag from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T23:31:09.663-0600] {logging_mixin.py:190} INFO - [2024-12-06T23:31:09.662-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-06T23:31:09.663-0600] {logging_mixin.py:190} INFO - [2024-12-06T23:31:09.663-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-06T23:31:09.663-0600] {processor.py:927} WARNING - No viable dags retrieved from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T23:31:09.680-0600] {processor.py:208} INFO - Processing /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.041 seconds
[2024-12-06T23:32:10.000-0600] {processor.py:186} INFO - Started process (PID=18021) to work on /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T23:32:10.007-0600] {processor.py:914} INFO - Processing file /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-06T23:32:10.008-0600] {logging_mixin.py:190} INFO - [2024-12-06T23:32:10.008-0600] {dagbag.py:588} INFO - Filling up the DagBag from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T23:32:10.014-0600] {logging_mixin.py:190} INFO - [2024-12-06T23:32:10.013-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-06T23:32:10.015-0600] {logging_mixin.py:190} INFO - [2024-12-06T23:32:10.014-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-06T23:32:10.015-0600] {processor.py:927} WARNING - No viable dags retrieved from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T23:32:10.034-0600] {processor.py:208} INFO - Processing /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.046 seconds
[2024-12-06T23:33:05.892-0600] {processor.py:186} INFO - Started process (PID=18096) to work on /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T23:33:05.899-0600] {processor.py:914} INFO - Processing file /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-06T23:33:05.901-0600] {logging_mixin.py:190} INFO - [2024-12-06T23:33:05.900-0600] {dagbag.py:588} INFO - Filling up the DagBag from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T23:33:05.907-0600] {logging_mixin.py:190} INFO - [2024-12-06T23:33:05.906-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-06T23:33:05.908-0600] {logging_mixin.py:190} INFO - [2024-12-06T23:33:05.908-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-06T23:33:05.908-0600] {processor.py:927} WARNING - No viable dags retrieved from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T23:33:05.926-0600] {processor.py:208} INFO - Processing /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.045 seconds
[2024-12-06T23:34:02.071-0600] {processor.py:186} INFO - Started process (PID=18166) to work on /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T23:34:02.078-0600] {processor.py:914} INFO - Processing file /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-06T23:34:02.081-0600] {logging_mixin.py:190} INFO - [2024-12-06T23:34:02.080-0600] {dagbag.py:588} INFO - Filling up the DagBag from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T23:34:02.087-0600] {logging_mixin.py:190} INFO - [2024-12-06T23:34:02.086-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-06T23:34:02.088-0600] {logging_mixin.py:190} INFO - [2024-12-06T23:34:02.087-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-06T23:34:02.088-0600] {processor.py:927} WARNING - No viable dags retrieved from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T23:34:02.108-0600] {processor.py:208} INFO - Processing /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.049 seconds
[2024-12-06T23:35:02.546-0600] {processor.py:186} INFO - Started process (PID=18293) to work on /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T23:35:02.553-0600] {processor.py:914} INFO - Processing file /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-06T23:35:02.555-0600] {logging_mixin.py:190} INFO - [2024-12-06T23:35:02.554-0600] {dagbag.py:588} INFO - Filling up the DagBag from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T23:35:02.561-0600] {logging_mixin.py:190} INFO - [2024-12-06T23:35:02.560-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-06T23:35:02.561-0600] {logging_mixin.py:190} INFO - [2024-12-06T23:35:02.561-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-06T23:35:02.562-0600] {processor.py:927} WARNING - No viable dags retrieved from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T23:35:02.578-0600] {processor.py:208} INFO - Processing /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.044 seconds
[2024-12-06T23:36:06.271-0600] {processor.py:186} INFO - Started process (PID=18373) to work on /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T23:36:06.278-0600] {processor.py:914} INFO - Processing file /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-06T23:36:06.280-0600] {logging_mixin.py:190} INFO - [2024-12-06T23:36:06.280-0600] {dagbag.py:588} INFO - Filling up the DagBag from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T23:36:06.286-0600] {logging_mixin.py:190} INFO - [2024-12-06T23:36:06.285-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-06T23:36:06.286-0600] {logging_mixin.py:190} INFO - [2024-12-06T23:36:06.286-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-06T23:36:06.286-0600] {processor.py:927} WARNING - No viable dags retrieved from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T23:36:06.304-0600] {processor.py:208} INFO - Processing /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.045 seconds
[2024-12-06T23:37:07.615-0600] {processor.py:186} INFO - Started process (PID=18559) to work on /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T23:37:07.616-0600] {processor.py:914} INFO - Processing file /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-06T23:37:07.618-0600] {logging_mixin.py:190} INFO - [2024-12-06T23:37:07.617-0600] {dagbag.py:588} INFO - Filling up the DagBag from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T23:37:07.621-0600] {logging_mixin.py:190} INFO - [2024-12-06T23:37:07.620-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-06T23:37:07.621-0600] {logging_mixin.py:190} INFO - [2024-12-06T23:37:07.621-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-06T23:37:07.621-0600] {processor.py:927} WARNING - No viable dags retrieved from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T23:37:07.630-0600] {processor.py:208} INFO - Processing /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.020 seconds
[2024-12-06T23:38:08.146-0600] {processor.py:186} INFO - Started process (PID=18777) to work on /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T23:38:08.153-0600] {processor.py:914} INFO - Processing file /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-06T23:38:08.155-0600] {logging_mixin.py:190} INFO - [2024-12-06T23:38:08.154-0600] {dagbag.py:588} INFO - Filling up the DagBag from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T23:38:08.160-0600] {logging_mixin.py:190} INFO - [2024-12-06T23:38:08.159-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-06T23:38:08.161-0600] {logging_mixin.py:190} INFO - [2024-12-06T23:38:08.161-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-06T23:38:08.161-0600] {processor.py:927} WARNING - No viable dags retrieved from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T23:38:08.181-0600] {processor.py:208} INFO - Processing /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.047 seconds
[2024-12-06T23:39:14.958-0600] {processor.py:186} INFO - Started process (PID=18923) to work on /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T23:39:14.960-0600] {processor.py:914} INFO - Processing file /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-06T23:39:14.962-0600] {logging_mixin.py:190} INFO - [2024-12-06T23:39:14.961-0600] {dagbag.py:588} INFO - Filling up the DagBag from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T23:39:14.965-0600] {logging_mixin.py:190} INFO - [2024-12-06T23:39:14.964-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-06T23:39:14.965-0600] {logging_mixin.py:190} INFO - [2024-12-06T23:39:14.965-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-06T23:39:14.966-0600] {processor.py:927} WARNING - No viable dags retrieved from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T23:39:14.985-0600] {processor.py:208} INFO - Processing /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.035 seconds
[2024-12-06T23:40:12.108-0600] {processor.py:186} INFO - Started process (PID=19215) to work on /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T23:40:12.109-0600] {processor.py:914} INFO - Processing file /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-06T23:40:12.110-0600] {logging_mixin.py:190} INFO - [2024-12-06T23:40:12.110-0600] {dagbag.py:588} INFO - Filling up the DagBag from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T23:40:12.112-0600] {logging_mixin.py:190} INFO - [2024-12-06T23:40:12.112-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-06T23:40:12.112-0600] {logging_mixin.py:190} INFO - [2024-12-06T23:40:12.112-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-06T23:40:12.112-0600] {processor.py:927} WARNING - No viable dags retrieved from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T23:40:12.122-0600] {processor.py:208} INFO - Processing /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.019 seconds
[2024-12-06T23:41:07.665-0600] {processor.py:186} INFO - Started process (PID=19311) to work on /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T23:41:07.666-0600] {processor.py:914} INFO - Processing file /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-12-06T23:41:07.667-0600] {logging_mixin.py:190} INFO - [2024-12-06T23:41:07.666-0600] {dagbag.py:588} INFO - Filling up the DagBag from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T23:41:07.671-0600] {logging_mixin.py:190} INFO - [2024-12-06T23:41:07.670-0600] {example_local_kubernetes_executor.py:40} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 38, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-12-06T23:41:07.671-0600] {logging_mixin.py:190} INFO - [2024-12-06T23:41:07.671-0600] {example_local_kubernetes_executor.py:41} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-12-06T23:41:07.671-0600] {processor.py:927} WARNING - No viable dags retrieved from /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-12-06T23:41:07.682-0600] {processor.py:208} INFO - Processing /home/m9d9v9/PycharmProjects/Airflow-2/Airflow-2/.venv/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
